modules/geo-location/class.jetpack-geo-location.php

```diff
@@ -249,7 +249,8 @@ public function register_rss_hooks() {
 	 * Add the georss namespace during RSS generation.
 	 */
 	public function rss_namespace() {
-		echo PHP_EOL . 'xmlns:georss="http://www.georss.org/georss" xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#"' . PHP_EOL;
+		echo PHP_EOL . "\t" . 'xmlns:georss="http://www.georss.org/georss\"';
+		echo PHP_EOL . "\t" . 'xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#"';
```

Comment:
What would you think about adding an extra line break and space after our additions, to play nice with other plugins that may come after us and add their own namespaces? This would avoid them issues like #9885 when other plugins come after us instead of before like in this issue.

```suggestion
		echo PHP_EOL . "\t" . 'xmlns:geo="http://www.w3.org/2003/01/geo/wgs84_pos#"';
		echo PHP_EOL . "\t";
```

Reply:
Sure, sounds good.